doc/user/ospfd.rst

```diff
@@ -824,10 +824,11 @@ Showing Information
 
    Show detailed information about the OSPF link-state database.
 
-.. clicmd:: show ip ospf route [json]
+.. clicmd:: show ip ospf route [detail] [json]
 
    Show the OSPF routing table, as determined by the most recent SPF
-   calculation.
+   calculation. If detail is specified, each routing item's
```

Comment:
Can you please correct the last sentence here. It is not clear what will be displayed if detail is specified in this statement. It will be better if you can mention it a little more clearly.

Reply:
You can mention something like, detail option can be used for more information like advertising router ID for the route, etc.

Reply:
OK.